api/ChattO/Application/DependencyInjection.cs

```diff
@@ -1,13 +1,19 @@
 ﻿using Microsoft.Extensions.DependencyInjection;
+using MediatR;
 using System.Reflection;
+using FluentValidation;
 
 namespace Application;
 
 public static class DependencyInjection
 {
     public static IServiceCollection AddApplication(
             this IServiceCollection services)
-    {   
+    {
+        services.AddMediatR(cfg => 
+            cfg.RegisterServicesFromAssemblies(AppDomain.CurrentDomain.GetAssemblies()));
```

Comment:
I think we can specify only "Application" assembly?

Reply:
Fixed